client/src/modules/cohort/index.jsx

```diff
@@ -277,15 +352,26 @@ class Cohort extends PureComponent {
       return null;
     }
 
-    const { isArchived, isOwner, name } = cohortDetails;
+    const {
+      externalAction,
```

Comment:
what is extrenalAction?

Reply:
Action that another user performed. These can be different actions. That's why the general name was given